cmd/run.go

```diff
@@ -142,6 +142,10 @@ func init() {
 	runCmd.Flags().StringVarP(&flgs.CacheDir, "cache-dir", "", "", flgs.Usage("CacheDir"))
 	runCmd.Flags().BoolVarP(&flgs.RetainCacheDir, "retain-cache-dir", "", false, flgs.Usage("RetainCacheDir"))
 	runCmd.Flags().StringVarP(&flgs.WaitTimeout, "wait-timeout", "", "10sec", flgs.Usage("WaitTimeout"))
+	runCmd.Flags().StringVarP(&flgs.EnvFile, "env-file", "", "", flgs.Usage("EnvFile"))
```

Comment:
```suggestion
	runCmd.Flags().StringVarP(&flgs.EnvFile, "env-file", "", ".env", flgs.Usage("EnvFile"))
```

The option name `env-file` reminded me of docker-compose and the default value seems to be `.env`!

Reply:
If there is a default value, it will always try to read that file due to the behaviour of Flags.
This is not the intended behaviour.

Reply:
https://docs.docker.com/compose/environment-variables/set-environment-variables/#substitute-with-an-env-file

> The .env file is the default method for setting environment variables in your containers. 

If you do not want the above to be the standard behavior, that is OK.

Reply:
nrhd...

I don't want to default load .env.

Because I don't think the .env file in the project root is for runn.

Thank you!!!

Reply:
There is room for debate, but I will keep it as it is for now.

Reply:
> Because I don't think the .env file in the project root is for runn.

Indeed, the .env in the repository root is for the application under test by runn.

You may use the same env content, or you may not want to use it.